src/components/pages/Manufacturers.jsx

```diff
@@ -0,0 +1,32 @@
+import { useSelector } from 'react-redux';
+import Header from '../Header';
+
+const Manufacturers = () => {
+  const manufacturers = useSelector((state) => state.manufacturers.manufacturers);
+  const manufacturersCount = useSelector((state) => state.manufacturers.count);
+
+  return (
+    <div className="manufacturers-page details-page">
+      <Header title="Manufacturers/Items" />
+      <h3>CATEGORY BREAKDOWN</h3>
+      <div className="details-header">
+        <h2>MANUFACTURERS</h2>
+        <p>{manufacturersCount}</p>
+      </div>
+      <h3>ITEMS</h3>
+      <ul>
+        {manufacturers.map((manufacturer) => (
+          <li key={manufacturer.Mfr_ID}>
+            {manufacturer.Mfr_Name}
+            <div className="more">
+              <p>{manufacturer.Mfr_ID}</p>
+              <i className="fa-regular fa-circle-right" />
+            </div>
+          </li>
+        ))}
+      </ul>
+    </div>
+  );
+};
+
```

Comment:
You did well with this app. Well, done 👏🏾 

- [ ] You added just too few tests. I did not see unit test for most of the components and some pure functions in the Redux code. You could take snapshots of each components. Also, I did not see an integration test using React Testing Library. Please add more tests.